src/OmniSharp/Utilities/AnalyzerAssemblyLoader.cs

```diff
@@ -0,0 +1,189 @@
+// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+#if DNX451
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Reflection.Metadata;
+using System.Reflection.PortableExecutable;
+using Roslyn.Utilities;
+
+namespace Microsoft.CodeAnalysis
+{
+    public class AnalyzerAssemblyLoader : IAnalyzerAssemblyLoader
```

Comment:
Nope. I saw a super simple one in the Roslyn tests, but it didn't work for analyzers that are spread across multiple assemblies.